### ISSUE TITLE
Addon does not work in ukrainian language.

### ISSUE THREAD

**Issue by Ishtef** [yplassiard/nvda-translate#14]
When NVDA interface language is set to ukrainian, addon does not work. Any text is translated into English. Please add the ability to use the application for users whose NVDA interface language is Ukrainian. Thanks.
